### ISSUE TITLE
Problem in building

### ISSUE THREAD

**Issue by mjshariati98** [jonhoo/simio#1]
I tried to build project using 'make' command but I got an error about getting go dependencies. I fix that by changing "code.google.com/p/gographviz" to "github.com/awalterschulze/gographviz" but now there is some errors in renderer.go file and I think It is because the library was changed from your last commits. Can you help me to build project ?

**Comment by jonhoo** [comment]
Yeah, it's probably due to changes in the `gographviz` library. Unfortunately I'm currently hard at work on my thesis, so I don't have much spare time to devote to this. If you share the compiler errors I could try to point you in the right direction?

**Comment by mjshariati98** [comment]
In the "renderer.go" file:

```go
func findEdge(network *graphviz.Graph, n1 string, n2 string) (uint, *graphviz.Edge) {
	n1e := network.Edges.SrcToDsts[n1]
	if n1e != nil && n1e[n2] != nil {
		return 0, n1e[n2]
	}

	n2e := network.Edges.SrcToDsts[n2]
	if n2e != nil && n2e[n1] != nil {
		return 1, n2e[n1]
	}

	return 0, nil
}
```

ERRORS :
```
src/renderer/renderer.go:33:16: cannot use n1e[n2] (type []*gographviz.Edge) as type *gographviz.Edge in return argument
src/renderer/renderer.go:38:16: cannot use n2e[n1] (type []*gographviz.Edge) as type *gographviz.Edge in return argument
```

according to "gographviz" package:
```go
type Edges struct {
	SrcToDsts map[string]map[string][]*Edge
	DstToSrcs map[string]map[string][]*Edge
	Edges     []*Edge
}
```

another ERROR is in the main function on this line: 
```go
network := graphviz.NewAnalysedGraph(tmpn).(*graphviz.Graph)
```
```
src/renderer/renderer.go:281:38: multiple-value gographviz.NewAnalysedGraph() in single-value context
```

where according to "gographviz" package : 
```go
// NewAnalysedGraph creates a Graph structure by analysing an Abstract Syntax Tree representing a parsed graph.
func NewAnalysedGraph(graph *ast.Graph) (*Graph, error) {
	g := NewGraph()
	if err := Analyse(graph, g); err != nil {
		return nil, err
	}
	return g, nil
}
```

Thanks.

**Comment by jonhoo** [comment]
Let's see. The first error should be solvable by making
```go
return 0, n1e[n2]
```
be
```go
return 0, n1e[n2][0]
```
(and similarly for the other `return`)

The second should be solved by making the relevant line:
```go
network, err := graphviz.NewAnalysedGraph(tmpn)
```

**Comment by mjshariati98** [comment]
Yes It's worked. Thanks a lot for your quick response. 
When I run demos, the graphical window doesn't open. I installed "python-pyinotify" package but does not work yet. only output a current.svg file in directory. Do you have any idea how I can fix it to have live view of simulation? In output there isn't any error, and just a warning: *Warning: missing an ID

**Comment by jonhoo** [comment]
Would you mind submitting a PR with those changes if that fixed your problem? That way others can benefit from this too :)

As for why the renderer doesn't work, I'm not quite sure. Does the `current.svg` contain anything? The renderer is Linux-only, so if you're trying to run it on another platform, that'd be reason. Other than that I don't know -- [`show.py`](https://github.com/jonhoo/simio/blob/master/src/renderer/src/renderer/show.py) is a really simple program, so I don't quite see how it'd be broken. Can you try just running it directly and passing it the path to an image file?

**Comment by mjshariati98** [comment]
I ran show.py according to your statement and I understood the problem is "pygtk" package and I installed it using 
sudo apt install python-gtk2 python-gtk2-dev
thanks for your help.

**Comment by jonhoo** [comment]
And that fixed the problem? That's great!

**Comment by mjshariati98** [comment]
Yes It is worked. Nice Simulator! :)

**Comment by jonhoo** [comment]
Thank you! I'm glad it worked out :) Would you mind filing a pull request with the changes you had to make so that others can benefit from the same in the future?

Also, out of interest, what are you using it for?

**Comment by mjshariati98** [comment]
I made a pull request. 
I want to use it in my "Distributed Systems" course project. It is about analyse Luby MIS algorithm in different typologies and under specific circumstances like node failures.

**Comment by jonhoo** [comment]
Fixed in #2.
